kuksa-sdk/src/test/kotlin/org/eclipse/kuksa/DataBrokerConnectionTest.kt

```diff
@@ -70,24 +72,16 @@ class DataBrokerConnectionTest : BehaviorSpec({
                     then("The #onPropertyChanged callback is triggered with the new value") {
                         val capturingSlot = slot<Types.DataEntry>()
 
-                        verify { propertyListener.onPropertyChanged(any(), any(), capture(capturingSlot)) }
+                        verify(timeout = 100) {
+                            propertyListener.onPropertyChanged(any(), any(), capture(capturingSlot))
+                        }
 
                         val dataEntry = capturingSlot.captured
                         val capturedDatapoint = dataEntry.value
                         val float = capturedDatapoint.float
 
                         assertEquals(newValue, float, 0.0001f)
                     }
-
-                    `when`("The same value is set again") {
```

Comment:
From what I remember this was a system test, when executing "feed Vehicle.Speed 100" twice, the observer was only triggered once. But for some reason this behavior suddenly changed? Funnily this test still ran through successfully on friday. 

Since it's no longer relevant I removed the test